_src/bundle.css

```diff
@@ -400,8 +400,32 @@ section h1{
 #campusMap img{
     max-width: 400px;
 }
-/** *********************************End Registraion page **/
+/** *********************************End Registration page **/
 
```

Comment:
@SavannahRicks At this point we just added some CSS specific for the local attractions page.